docs/sources/alerting/fundamentals/alert-rules/queries-conditions.md

```diff
@@ -41,138 +36,154 @@ refs:
+- A [query](#data-source-queries) that specifies the data to retrieve from a data source.
+- A [condition](#alert-condition) that must be met before the alert rule fires.
+- Optional [expressions](#expressions-advanced-options) to perform transformations on the retrieved data.
+
+Alerting periodically runs the queries and expressions, evaluating the condition. If the condition is breached, an alert instance is triggered for each time series.
 
 ## Data source queries
 
-Alerting queries are the same type of queries available in Grafana panels. Queries in Grafana can be applied in various ways, depending on the data source and query language being used. However, not all [data sources support Alerting](ref:data-source-alerting).
+Alerting queries are the same as the queries used in Grafana panels.
+
+However, Grafana-managed alerts can only query [backend data sources with Alerting enabled](ref:data-source-alerting).
 
-Each data source’s query editor provides a customized user interface to help you write queries that take advantage of its unique capabilities. For additional information about queries in Grafana, refer to [Query and transform data](ref:query-transform-data).
+Queries in Grafana can be applied in various ways, depending on the data source and query language being used. Each data source’s query editor provides a customized user interface to help you write queries that take advantage of its unique capabilities.
+
+For more details about queries in Grafana, refer to [Query and transform data](ref:query-transform-data).
+
+{{< figure src="/media/docs/alerting/alerting-query-conditions-default-options.png" max-width="750px" caption="Define alert query and alert condition" >}}
+
+## Alert condition
 
-Some common types of query components include:
+The alert condition is the query or expression that determines whether the alert fires or not depending whether the value satisfies the specified comparison. There can be only one condition which determines the triggering of the alert.
 
-**Metrics or data fields**: Specify the specific metrics or data fields you want to retrieve, such as CPU usage, network traffic, or sensor readings.
+If the queried data meets the defined condition, Grafana fires the alert.
 
-**Time range**: Define the time range for which you want to fetch data, such as the last hour, a specific day, or a custom time range.
+When using **Default options**, the `When` input reduces the query data, and the last input defines the threshold condition.
 
-**Filters**: Apply filters to narrow down the data based on specific criteria, such as filtering data by a specific tag, host, or application.
+When using **Advanced options**, you have to choose one of your queries or expressions as the alert rule condition.
 
-**Aggregations**: Perform aggregations on the data to calculate metrics like averages, sums, or counts over a given time period.
+## Expressions (Advanced options)
```

Comment:
Wondering whether to have this the other way around Advanced options: Expressions?

Reply:
fixed by https://github.com/grafana/grafana/pull/95109/commits/960b783e48550cac776d37d41ae132fb6543eaa7